modular_skyrat/modules/mapping/voidraptor/code/turfs.dm

```diff
@@ -0,0 +1,4 @@
+/turf/open/floor/vault/mundane
+	name = "floor"
+	desc = null
+	floor_tile = /obj/item/stack/tile/iron/base
```

Comment:
What's the point of this, exactly?

Reply:
So it has a normal name instead of weird one, no description and drops a floor tile, I can just make it a subtype of iron floor if you prefer, does not make a difference.

Reply:
Yeah but like, why does it need to not have a name, if it's the standard for vaults to be like that?

Reply:
The normal one is used sparsely on maps and called "strange floor" with a desc "You feel a strange nostalgia from looking at this..."

I have the whole command area plated with it for the unique sprite and look. Its just a normal floor otherwise and this returns the name and desc to such, also makes the tile not poof.

Reply:
Floor removed and some places lil remapped